app/views/govuk_publishing_components/components/docs/list.yml

```diff
@@ -0,0 +1,59 @@
+name: List
+description: A list, unordered or ordered.
+body: |
+  This is an ordered or unordered list, with or without visible bullets / numbers.
+accessibility_criteria: |
+  The list must:
+
+  - inform the user how many items are in the list
+  - convey the content structure
+  - indicate the current page when contents span different pages, and not link to itself
+
+  The contents list may:
+
+  - include an aria-label to contextualise the list if helpful
+shared_accessibility_criteria:
+  - link
```

Comment:
Is this required? The component itself doesn't contain links, unless they're passed to it. I guess since that's allowed this is okay, but then don't we need other accessibility criteria to cover anything else that might be passed? I'd rather keep it simple. Maybe just a criteria to say anything passed should meet accessibility criteria for itself. Or something.

Reply:
Good point - I've removed `link`